pallets/parachain-staking/src/lib.rs

```diff
@@ -438,6 +441,31 @@ pub mod pallet {
 				weight = weight.saturating_add(T::OnNewRound::on_new_round(round.current));
 				// pay all stakers for T::RewardPaymentDelay rounds ago
 				weight = weight.saturating_add(Self::prepare_staking_payouts(round.current));
+
+				// select total candidates
+				let candidates = <CandidatePool<T>>::get().0;
+
+				// iter candidates to check which of them must be marked as offline
+				for candidate in candidates {
+					match <CandidateRoundsInfo<T>>::try_get(candidate.owner.clone()) {
```

Comment:
Why try_get? Shouldnt this return an option when you do get (since its an option query)?

Reply:
`<CandidateRoundsInfo<T>>::get(&candidate.owner).unwrap_or_default()` should be sufficient to detect the inactive collators

Reply:
Yes `get()` is enough because it's an OptionQuery map